src/node_report_utils.cc

```diff
@@ -77,7 +78,7 @@ static void ReportEndpoints(uv_handle_t* h, JSONWriter* writer) {
     default:
       break;
   }
-  ReportEndpoint(h, rc == 0 ? addr : nullptr, "remoteEndpoint", writer);
+  ReportEndpoint(h, rc == 0 ? addr : nullptr, "remoteEndpoint", writer, exclude_network);
```

Comment:
Is it normal that this function is executing 2 times the exact same thing with only the key name that differs? Maybe we can optimise it as well for report with network so that the query is done only once and not twice